src/sip.c

```diff
@@ -354,15 +423,27 @@ sip_check_packet(packet_t *packet)
     // Get Method and request for the following checks
     // There is no need to parse all payload at this point
     // If no response or request code is found, this is not a SIP message
-    if (!sip_get_msg_reqresp(msg, payload)) {
+    if(channelid[0]) {
+        if(!sip_get_msg_reqresp_for_mrcp(msg, payload)) {
+            printf("bad bad\n");
+            exit(1);
```

Comment:
I think this is debugging code, you should not exit sngrep from capture layer.

Reply:
Yes. Sorry about that.